docs/new/index.md

```diff
@@ -6,15 +6,32 @@ At Percona, we've been hard at work listening to your feedback and developing in
 
 ## Featured announcements 
 
+
 <div data-grid markdown>
+<div data-banner="pmm" markdown>
+
+### <span style="font-size:1.875em;margin-right:0.125em">:percona-pmm:</span> Percona Monitoring and Management 3.0.0 General Availability {.title}
+<br>
+We’re excited to announce the General Availability (GA) of PMM 3.0.0, a new major version that introduces a complete architectural overhaul of PMM.
+
+
+<div class="actions" markdown>
+
+[Read more :material-arrow-right:](../new/posts/Percona%20Monitoring%20and%20Management/pmm-3.0.0-release.md){:target="_blank"}
+
+<span style="float: right;">January 30, 2025</span>
+
+</div>
+</div>
+
 <div data-banner="everest" markdown>
```

Comment:
This piece still relates to Everest. It must be changed to PMM

Reply:
@nastena1606  I think that one is for Everest's entry below